Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -34,15 +34,15 @@ public class LoudObject : MonoBehaviour
     void Start()
     {
         rb = GetComponent<Rigidbody>();
-        rends = GetComponents<Renderer> ();
+        rends = GetComponents<Renderer>();
         plateShader = Shader.Find(Constants.SHADER_NAME_SHAKE);
         noisePing = gameObject.GetComponent<NoisePinger>();
     }
 
     void Update()
     {
         distance = Vector3.Distance(transform.position, GameManager.Instance.GetPlayerTransform().position);
-
+        thrustDirection = -GameManager.Instance.GetPlayerTransform().forward;

```

Comment:
Maybe can we just remove this variable entirely, and just use this value within the if statement below? Inside the AddForce call? Would be cleaner that way.